managers.md

```diff
@@ -1,115 +1,107 @@
+- You own overall retention strategy on the team.
+### Software Engineering & Design
+
+- You ensure the team has strong technical best practices. You provide the team with opportunities to enhance technical skills.
+- You are a technical and interpersonal role model. You help resolve technical issues and advise on technical decisions.
+
+### Execution & Results
+
+- You define processes (e.g. grooming, retro, team syncs, etc) for the team with cross-functional counterparts and the team directly.
+- You ensure all members of your team have the opportunity to demonstrate project ownership. You may opt to take project management on for high stakes or high urgency initiatives.
+- You shield the team from external changes (to the degree possible). You keep the team focused on the tasks explicitly assigned. You influence the decisions that are made regarding your team directly.
+
+### Collaboration & Communication
+
+- You partner with relevant counterparts (e.g. Product, Design, Marketing, Curriculum, Data Science, etc…) to build team roadmaps, establish best practices.
+- You escalate challenges the team is facing to the larger engineering (or engineering leadership) team, and suggest solutions.
+
+### Community & Citizenship
+
+- You own the success of your entire team. You ensure the team is celebrated for successes and has opportunities to learn from failure. 
+- You start to own at least one engineering-wide initiative (e.g. tech community pillar, interviewing, revising competencies)
+
+## Senior Engineering Manager
+
+You are an experienced manager who manages up to two teams directly or
+through managing a manager.
+
+### Management & Leadership
+
+- You manage up to two teams either directly or through managing a manager within the same Organization.
+- You manage & coach ICs. You look for opportunities to grow or manage new engineering managers (EMs) or Staff Engineers. You have multiple instances of successful promos.
+- You plan for team needs through suggesting headcount, proposing tools & integrations, and identifying dependencies. You understand what needs other teams within the same Organization have, and escalate or adjust if those needs will impede your team’s needs. You understand the strategy for the Organization, and ensure your team is lined up to support it.
+- You scout and attract talent. You build relationships with potential candidates. You influence org wide hiring and staffing. You moderate hiring decisions.
+- You identify macro-level patterns of team engagement, influence decisions on retention.
 
 ### Software Engineering & Design
 
-- You hold your team to the software engineering and design standards we have set for individual contributors.
-- You provide an environment (process, tools, goals, sprints, etc) that allows individual contributors to do their best work.
-- When you contribute technically, you exemplify the standards we set for individual contributors.
```

Comment:
Why remove the first and third items from this competency? Those still seem relevant